CorsixTH/Lua/languages/japanese.lua

```diff
@@ -0,0 +1,2440 @@
+      [4] = "驚くべき病院経営です。招待してくださって感謝します。",
+      [5] = "まあ、悪くはないですな...",
+      [6] = "良い病院ですな。ところで皆さん、タージでカレーでもいかがです？",
+      [7] = "ふむ、もっとひどい所も見て来ましたが...改善が必要なのは言うまでもないですな。",
+      [8] = "いやいや、気分が悪くなっても、ここだけは来たくないですな。",
+      [9] = "率直に言って何の変哲もない病院ですな。もっと期待していたんですが。",
+      [10] = "なんでわざわざ来たんだろう？４時間連続のオペラの方がマシだ！",
+      [11] = "目にしたものにがっかりしたよ。これを病院と呼ぶのか？豚小屋と言った方がお似合いだな！",
+      [12] = "公の場で注目を浴び、こんな臭い場所を訪れさせられることにうんざりだ！もうこんな仕事辞めてやる。",
+      [13] = "なんてひどい場所だ。ここを閉鎖するために運動を開始する！",
+      [14] = "こんなに酷い病院は見たことがない。なんてざまだ！",
+      [15] = "正直ショックを受けました。あれは病院と呼べる代物じゃない。一杯引っかけに行くぞ！",
+      bad = {
+        [1] = "なんでわざわざ来たんだろう？４時間連続のオペラの方がマシだ！",
+        [2] = "目にしたものにがっかりしたよ。これを病院と呼ぶのか？豚小屋と言った方がお似合いだな！",
+        [3] = "公の場で注目を浴び、こんな臭い場所を訪れさせられることにうんざりだ！もうこんな仕事辞めてやる。",
+      },
+      mediocre = {
+        [1] = "ふむ、もっとひどい所も見て来ましたが...改善が必要なのは言うまでもないですな。",
+        [2] = "いやいや、気分が悪くなっても、ここだけは来たくないですな。",
+        [3] = "率直に言って何の変哲もない病院ですな。もっと期待していたんですが。",
+      },
+      super = {
+        [1] = "なんて素晴らしい病院だ。次に本当に具合が悪くなったら、絶対にここに入院するぞ。",
+        [2] = "これこそが本当の病院だ！",
+        [3] = "これは素晴らしい病院だ。いくつかの病院に入ったことがあるからわかりますよ。",
+      },
+      very_bad = {
+        [1] = "なんてひどい場所だ。ここを閉鎖するために運動を開始する！",
+        [2] = "こんなに酷い病院は見たことがない。なんてざまだ！",
+        [3] = "正直ショックを受けました。あれは病院と呼べる代物じゃない。一杯引っかけに行くぞ！",
+      },
+      free_build = {
+        [1] = "とても素晴らしい病院ですね！ただ、資金の制限がなければ運営するのはそれほど難しくないですよね？",
+        [2] = "私は経済学者ではありませんが、私でもこの病院を運営できると思います。この意味わかりますよね？",
+        [3] = "とてもよく運営されている病院です。ただ、不況には気をつけてくださいね！…そうか、あなたには関係ない話でしたね。",
+      },
+      good = {
+        [1] = "驚くべき病院経営です。招待してくださって感謝します。",
+        [2] = "まあ、悪くはないですな...",
+        [3] = "良い病院ですな。ところで皆さん、タージでカレーでもいかがです？",
+      },
+    },
+    rep_boost = "この地域でのあなたの評判が向上しました。",
+    telegram = "電報！",
+  },
+  disease_discovered_patient_choice = {
+    guessed_percentage_name = "チームは患者の病状を推測する必要がありました。それが%sである確率は%dパーセントです。",
+    need_to_employ = "この状況に対処するためには%sを雇用してください。",
+    disease_name = "研究チームは新しい病気「%s」を発見しました。",
+    choices = {
+      wait = "患者をしばらく病院で待たせる。",
+      send_home = "患者を自宅へ送り返す。",
+      research = "患者を研究科に送る。",
+    },
+    need_to_build = "これに対処するためには%sを建設する必要があります。",
+    need_to_build_and_employ = "%sを設置し、%sを雇用すればうまくいきます。",
+    can_not_cure = "この病気を治療することはできません。",
+    what_to_do_question = "患者をどうしますか？",
+  },
+  epidemic_result = {
+    rep_loss_fine_amount = "新聞がこの伝染病事件を大きく取り上げるでしょう。あなたの評判は大きく傷つき、さらに罰金として$%dが科されます。",
+    succeeded = {
+      part_1_name = "検査官があなたの病院で%sが広まったという噂を聞きつけました。",
+      part_2 = "しかしながら、検査官は噂を裏付ける証拠を見つける事ができませんでした。",
+    },
+    close_text = "やった！",
+    hospital_evacuated = "健康委員会はあなたの病院を閉鎖する以外、手がありませんでした。",
+    fine_amount = "政府は国家緊急事態を宣言し、罰金として%dが科されます。",
+    compensation_amount = "政府はこの噂があなたの病院に与えた損害の代償金として$%dを授与することを決定しました。",
+    failed = {
+      part_1_name = "伝染病%sの発生を隠そうとした結果、",
+      part_2 = "あなたの病院スタッフは地域に伝染病を広げてしまいました。",
+    },
+  },
+}
+
+vip_names = {
```

Comment:
With regards to the VIP names, you are more than welcome to replace these direct translations with localised variants, such as parodies of national celebrities or politicians.